tools/bioformats2raw/bf2raw.xml

```diff
@@ -0,0 +1,127 @@
+<tool id="bf2raw" name="bioformats2raw" version="0.0.1">
```

Comment:
According to https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=version#tool-versions:
```suggestion
<tool id="bf2raw" name="bioformats2raw" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
```

Running `bioformats2raw --version` inside the container tells me that it is version 0.7.0, so according to https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=version#tool-versions, you also need to add:
```xml
    <macros>
         <token name="@TOOL_VERSION@">0.7.0</token>
         <token name="@VERSION_SUFFIX@">0</token>
     </macros>
```

Reply:
This is done now.